content/docs/buildpack-basics.md

```diff
@@ -0,0 +1,63 @@
++++
+title="Basics of BuildPacks"
+weight=2
+getting-started=true
++++
+
+## Basic Concepts
+
+### What is a Buildpack?
+
+A `buildpack` is software that transforms application source code into
+executable files by analyzing the code and determining the best way to
+build it.
+
+![buildpacks](/images/what.svg)
+
+### What is a Builder?
+
+A builder is an image that contains all the components necessary to
+execute a build (for example, an ordered combination of buildpacks, a build
+image and other files and configurations).
+
+![create-builder diagram](/images/create-builder.svg)
+
+### What is a Lifecycle?
+
+A lifecycle is a series of steps that are used to create and manage a
+buildpack. `create` is used to analyze, detect, restore, build, and export
+buildpack execution. Next, `launcher` can be used to launch the application.
+Finally, `rebase` can be used to push the latest changes to an existing
+buildpack. All of these steps are part of a lifecycle.
+
+![lifecycle](/images/lifecycle.png)
+
+### What is a Platform
+
+A platform typically refers to an organization or service provider (e.g.,
+kpack, Tekton, Fly.io, Digital Ocean, Google Cloud, Heroku, SalesForce, etc.)
+that incorporates Buildpacks within their products to make buildpack
+functionality available to their end-users (typically, application
+developers). 
```

Comment:
In the buildpacks project we have a clear idea of what a platform is (in our platform spec).  I wonder if we need to tease apart the idea of a platform and a platform oprerator in this paragraph?

Reply:
- Yeah, I wrestled with this during the first draft, I guess my over-simplification changed the meaning. How about the following:

### What is a Platform:

_A platform coordinates builds by invoking the lifecycle binary together with the buildpacks and the application source code in order to produce an executable OCI image_.

### Platform Operators

_Platform Operators are organizations or service providers (e.g., kpack,
Tekton, Fly.io, Digital Ocean, Google Cloud, Heroku, SalesForce, etc.) that
incorporates Buildpacks within their products to make buildpack functionality
available to their end-users (typically, application developers)._